### ISSUE TITLE
Revisit requireFail color and message

### ISSUE THREAD

**Issue by phated** [gulpjs/gulp-cli#142]
It seems that some consumers are allergic to the color red - we should revisit the coloring and messaging of the `requireFail` log.

Ref https://github.com/zurb/foundation-sites/issues/10828
and https://github.com/gulpjs/gulp/issues/1631

**Comment by ncoden** [comment]
I am really impressed by how close you are to the `gulp` community. Thank you and 👍 for this issue.

**Comment by phated** [comment]
We might also want to change this to a `log.warn`

**Comment by njnygaard** [comment]
I'm allergic to the color red because it gets passed through to my docker build. It is not a failure, just attention grabbing. Just as a note, npm warnings about "EINTEGRITY" come through as 'red' too in a docker build. And what I mean by that is `log.warn` might not quiet those with the allergy.

**Comment by phated** [comment]
@sttk I just realized this issue wasn't fully completed by my PR.  We still need to change the wording of the messages.   I also think a "beforeRequire" event in liftoff would help improve messaging.  Thoughts?

**Comment by phated** [comment]
@njnygaard I'm guessing you don't know of our --no-color flag?

**Comment by contra** [comment]
Why are we logging this message at all if it ends up working?

**Comment by phated** [comment]
@contra some of the logs happen in realtime (require logged right after it's required or fails) while some is batched but we don't know which is which.  Also, I think there was a desire to show the user when they were using an older loader (e.g. `babel-register` instead of `@babel/register`)

**Comment by sttk** [comment]
@phated 
> I also think a "beforeRequire" event in liftoff would help improve messaging. 

Yes, I remember that there is a request to add the event to liftoff (https://github.com/js-cli/js-liftoff/issues/53). I agree it, and I'll address it if I can.

**Comment by moos** [comment]
Fails completely with node `v10.1.0` -- unable to proceed.

```
> gulp start

[20:48:42] Failed to load external module @babel/register
[20:48:42] Requiring external module babel-register
gulp[38618]: ../src/node_contextify.cc:628:static void node::contextify::ContextifyScript::New(const FunctionCallbackInfo<v8::Value> &): Assertion `args[1]->IsString()' failed.
 1: node::Abort() [/Users/willywonka/dev/homebrew/bin/node]
 2: node::Assert(char const* const (*) [4]) [/Users/willywonka/dev/homebrew/bin/node]
 3: node::contextify::ContextifyScript::New(v8::FunctionCallbackInfo<v8::Value> const&) [/Users/willywonka/dev/homebrew/bin/node]
 4: v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo*) [/Users/willywonka/dev/homebrew/bin/node]
 5: v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<true>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [/Users/willywonka/dev/homebrew/bin/node]
 6: v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/Users/willywonka/dev/homebrew/bin/node]
 7: 0x6bfa608427d
```

**Comment by phated** [comment]
This was closed by #182 because it pulled in Liftoff 3.1, and v3 removed the extra logging if we found 1 successful loader.

It should be released very soon.
